exercises/list-ops/list-ops.spec.js

```diff
@@ -0,0 +1,95 @@
+var list = require('./list-ops');
+
+describe('List', function () {
+  describe('append', function () {
+    xit('appends two empty lists', function () {
+      expect(list.append([], [])).toEqual([]);
+    });
+
+    xit('appends an empty list to a non-empty list', function () {
+      expect(list.append([], [1, 2, 3, 4])).toEqual([1, 2, 3, 4]);
+    });
+
+    xit('appends two non-empty lists', function () {
+      expect(list.append([1, 2], [2, 3, 4, 5])).toEqual([1, 2, 2, 3, 4, 5]);
+    });
+  });
+
+  describe('concat', function () {
+    xit('concatenates an empty list', function () {
+      expect(list.concat([])).toEqual([]);
+    });
+
+    xit('concatenates a list of lists', function () {
+      expect(list.concat([[1, 2], [3], [], [4, 5, 6]])).toEqual([1, 2, 3, 4, 5, 6]);
+    });
+  });
+
+  describe('filter', function () {
+    var isOdd = x => x % 2 === 1;
```

Comment:
In this line (as well as `51` and `63`) makes use of fat arrow functions.

That's a language feature that's been released... 2 years ago? I'm not sure. Anyway, that language features are more appropriate for the `ecmascript` track. This track should stick to *old* ES5 JavaScript features.

It wouldn't  be a problem for the `example.js` file, but tests are going to be visible for users. They should not expect ES6 (or newer) features in exercises of this track.

It's not a problem with the code, it's a problem with the philosofy/spirit of `javascript` and `ecmascript` tracks.